components/vfs/vfs.c

```diff
@@ -33,15 +33,14 @@
 
 static const char *TAG = "vfs";
 
-#define VFS_MAX_COUNT   8   /* max number of VFS entries (registered filesystems) */
```

Comment:
One note, here I had to re-introduce VFS_MAX_COUNT macro so it's defined even if CONFIG_VFS_SUPPORT_IO is not set. Otherwise the build would fail when CONFIG_VFS_SUPPORT_IO is disabled.

Reply:
Thanks for your time and sorry for wasting it on this not fully functional pull request!

I'm now instead following a pattern I find all over the place in the ESP-IDF code base.